addons/lunch/wizard/lunch_confirm.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, api
+
+
+class lunch_confirm_order(models.TransientModel):
```

Comment:
Is this wizard really necessary ? A confirm button directly on the form view is not sufficient ? I don't think we have to prompt wizard on every action ...